tests/TestHelpers/GraphHelper.php

```diff
@@ -1532,4 +1539,88 @@ public static function getPermissionsList(
 			self::getRequestHeaders()
 		);
 	}
+
+	/**
+	 * Get the role id by name
+	 *
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $role
+	 *
+	 * @return string
+	 *
+	 */
+	public static function getRoleIdByName(
```

Comment:
We have two roles with different names, that's right.
this method doesn't return second role 
https://github.com/owncloud/ocis/issues/7995#issuecomment-1860200207